files/sdw-admin.py

```diff
@@ -80,16 +84,124 @@ def copy_config():
         raise SDWAdminException("Error copying configuration")
 
 
+def provision_and_configure():
+    """
+    Applies the salt state.highstate on dom0 and all VMs
+    """
+    provision("Provisioning Fedora-based system VMs", "securedrop_salt.sd-sys-vms")
+    provision("Provisioning base template", "securedrop_salt.sd-base-template")
+    configure("Configuring base template", ["sd-base-bookworm-template"])
+    provision_all()
+    configure("Configuring logging VMs early", ["sd-small-bookworm-template"], restart=["sd-log"])
```

Comment:
Nit: suggest clarifying, something like:

```suggestion
    configure("Configuring template for log sink before anything else", ["sd-small-bookworm-template"], restart=["sd-log"])
```

But feel free to ignore this if you want to keep this pull request as a strict port.